tests/python_package_test/test_dask.py

```diff
@@ -19,10 +19,10 @@
 import pandas as pd
 from scipy.stats import spearmanr
 from dask.array.utils import assert_eq
+from dask.distributed import wait
 from distributed.utils_test import client, cluster_fixture, gen_cluster, loop
 from scipy.sparse import csr_matrix
 from sklearn.datasets import make_blobs, make_regression
-from sklearn.utils import check_random_state
```

Comment:
oh thanks for removing this. Didn't realize it was unused.

Reply:
all credit goes to pycharm